jerry-core/config.h

```diff
@@ -80,7 +85,7 @@
  * Also the option affects size of ECMA Object Model's data types.
  * In any case size of any of the types should not exceed CONFIG_MEM_POOL_CHUNK_SIZE.
  */
-#define CONFIG_ECMA_REFERENCE_COUNTER_WIDTH (10)
+#define CONFIG_ECMA_REFERENCE_COUNTER_WIDTH (9)
```

Comment:
With the increased heap size, we now need to use full 16 bit compressed pointers, while previously 15 bits were enough to address the entire heap. The current object model requires it to fit into 64 bits, unfortunately this meant that I needed to cut those bits from somewhere else. We could keep it the way it was, but we then have to keep the heap at 256kB as well.